__tests__/database/migration/file/migration_version_5.test.ts

```diff
@@ -0,0 +1,33 @@
+import MigrationVersion5 from '../../../../src/database/migration/file/migration_version_5';
+import {mockDb} from '../../../../__utils__/mockedVariable/mockDb';
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('TESTING Migration Version 5', () => {
+  it('TEST migration up function', async () => {
+    // Setup
+
+    // Execution
+    MigrationVersion5.up(mockDb);
+
+    // Assertion
+    expect(mockDb.executeSql).toHaveBeenCalledWith(
+      `ALTER TABLE channel_lists
+      ADD COLUMN expired_at DATETIME DEFAULT NULL;`
+    );
+  });
+
+  it('TEST migration down function', async () => {
+    // Setup
+
+    // Execution
+    MigrationVersion5.down(mockDb);
+
+    // Assertion
+    expect(mockDb.executeSql).toHaveBeenCalledWith(
+      'ALTER TABLE channel_lists DROP COLUMN expired_at'
+    );
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `down` function test is missing an assertion to check if the column `expired_at` exists before attempting to drop it. This could lead to errors if the column does not exist. Consider adding a check for the existence of the column before dropping it.
```diff
+    // Setup
+    const columnExists = await mockDb.executeSql('PRAGMA table_info(channel_lists);');
+    const hasExpiredAtColumn = columnExists.some(column => column.name === 'expired_at');
+
+    // Execution
+    if (hasExpiredAtColumn) {
+      MigrationVersion5.down(mockDb);
+    }
```
Please note that this is a simplified example and may need to be adjusted based on your actual database interaction library and its syntax.


<!-- This is an auto-generated comment by OSS CodeRabbit -->